CRM/Dedupe/Merger.php

```diff
@@ -619,19 +619,19 @@ public static function retrieveFields($main, $other) {
    *
    * @return array|bool
    */
-  public static function batchMerge($rgid, $gid = NULL, $mode = 'safe', $batchLimit = 1, $isSelected = 2, $criteria = array(), $checkPermissions = TRUE) {
```

Comment:
batchMerge is never actually called without $batchLimit being passed so we can change or alter the default. Coin toss as to which but I came down on changing to 0 which means no limit